coriolis/osmorphing/manager.py

```diff
@@ -123,7 +123,13 @@ def morph_image(origin_provider, destination_provider, connection_info,
         if packages_add:
             event_manager.progress_update(
                 "Adding packages: %s" % str(packages_add))
-            import_os_morphing_tools.install_packages(packages_add)
+            try:
+                import_os_morphing_tools.install_packages(
+                    packages_add)
+            except Exception as err:
+                raise exception.CoriolisException(
+                    "Failed to install packages: %s" % ", ".join(
```

Comment:
Considering we're hiding the real error message in the Replica view, I'd also add a "Please review the logs for more details" to the message

Reply:
Was actually thinking about this. Adding.